db/re/status.yml

```diff
@@ -9132,3 +9132,12 @@ Body:
       NoDispell: true
       NoBanishingBuster: true
       NoClearance: true
+  - Status: Mystery_Powder
+    Icon: EFST_MYSTERY_POWDER
+    DurationLookup: BO_MYSTERY_POWDER
+    Flags:
+      BlEffect: true
```

Comment:
Not required
```suggestion
```
Unknown if `DisplayPc` flag is set, if anyone have this info

Reply:
Deleting this gonna remove the status effect